lib/ui/fixtures/ui_test.dart

```diff
@@ -288,3 +287,354 @@ Future<void> pumpImage() async {
 }
 void _captureImageAndPicture(Image image, Picture picture) native 'CaptureImageAndPicture';
 Future<void> _onBeginFrameDone() native 'OnBeginFrameDone';
+
+@pragma('vm:entry-point')
+void hooksTests() {
+  void test(String name, VoidCallback testFunction) {
+    print(name);
```

Comment:
can we avoid printing unless the test fails? (ideally tests would have no output except to report failures or to say that all the tests passed)

Reply:
Done